src/core/qgssqliteutils.cpp

```diff
@@ -243,7 +247,8 @@ long long QgsSqliteUtils::nextSequenceValue( sqlite3 *connection, const QString
     }
   }
 
-  dsPtr.release();
+  // INTENTIONAL HACK -- see above
+  ( void )dsPtr.release();
```

Comment:
I'm not finding in the coverity database this issue, but you can potentially write a suppression, presumably:
```suggestion
  // coverity[leaked_storage]
  ( void )dsPtr.release();
```

Reply:
If the void cast fixes the warning, is there a need for a specific coverity suppression?

Reply:
> If the void cast fixes the warning, is there a need for a specific coverity suppression?

no indeed (I would be surprised that the cast fixes the warning, but perhaps I'm wrong)